companies/jpmorgan-chase.md

```diff
@@ -0,0 +1,11 @@
+---
+fortune_rank:17
+name: JPMorgan Chase
+slug: jpmorgan-chase
+blm_statements:
+  - url:
```

Comment:
```suggestion
  - url: https://www.jpmorganchase.com/news-stories/message-from-jamie-dimon-and-brian-lamb
```

Reply:
That link did not specifically address BLM, should it still be on line 6?

Reply:
Thank you for pointing this out!  This makes me realize that our team needs to be a bit more clear about what we are collecting.  We are looking for corporate statements in response to the murder of George Floyd, which may or may not explicitly say 'Black Lives Matter'.  This one is from May 29 and talks about 'terrible events in Minneapolis' so I would like to include it. :)